blocks/editable/index.js

```diff
@@ -45,11 +45,11 @@ function createTinyMCEElement( type, props, ...children ) {
 		return children;
 	}
 
-	return createElement(
+	return [ [
```

Comment:
Ideally we should get rid of these double arrays... Should we still take it through Dom-react at all? See https://github.com/WordPress/gutenberg/pull/3048#issuecomment-337913778

Reply:
Does it mean that we should implement`nodeListToReact` ourselves?